ionic2/app/pages/search/search.page.ts

```diff
@@ -64,7 +64,7 @@ export class SearchPage {
 
   selectPokemon(pokemon:Pokemon) {
     this.cancelRequests();
-    this.navCtrl.setRoot(WikiPage, {pokemonId: pokemon.pokemonId});
+    this.navCtrl.setRoot(PokeDetailPage, {pokemonId: pokemon.pokemonId});
```

Comment:
I know it's my code but at the time of writing we didn't have both the PokeDetailsPage and WikiIndexPage yet :D Thanks for changing it!